addons/project_timesheet_holidays/views/hr_holidays_views.xml

```diff
@@ -9,7 +9,7 @@
             <xpath expr="//group[@name='visual']" position="before">
                 <group name="timesheet" groups="base.group_no_one" string="Timesheet">
                     <div class="text-muted mb-4" colspan="2">
-                        Generate timesheets when validating time off of this type.
+                        Generate timesheets for validated time offs and public holidays
```

Comment:
```suggestion
                        Generate timesheets when validating time off requests of this type 
```
For this, I spoke with Alexandra and she would prefer to leave the description as is but with this small modification (having "time off requests" instead of "time off")